openrtx/include/calibration/calibInfo_GDx.h

```diff
@@ -76,16 +76,16 @@ typedef struct
     uint16_t mixGainNarrowband;        // 0x06A
     uint8_t rxDacGain;                 // 0x06C
     uint8_t rxVoiceGain;               // 0x06D
-}
-bandCalData_t;
+} bandCalData_t;
 
 typedef struct
 {
-    bandCalData_t data[2];      // Calibration data for VHF (index 0) and UHF (index 1) bands
-    freq_t vhfCalPoints[8];     // VHF calibration points for both TX power and mod1Amplitude
+    bandCalData_t
+        data[2]; // Calibration data for VHF (index 0) and UHF (index 1) bands
```

Comment:
Is there a way to avoid having the lint doing this? Ok that the line goes above the 80 column limit, but breaking it like this make it unreadable. Probably the only way to fix this behavior is to split the comment (but geez).